### PR TITLE
Layout demo

### PR DESCRIPTION
**Pull request by leejordan**
Final examples for feature blocks after lots of design review

### CODE REVIEW

File: less/common/typography.less [hunk 1]
```diff
@@ -9,7 +9,7 @@ small {
 h1, h2, h3, h4, h5, h6 {
     .lowercase();
     font-weight: @font-weight-heading;
-    .margins(2, 0, 0.5, 0);
+    .margins(1, 0, 0.5, 0);
```
Comment (notlee) [inline less/common/typography.less]: This seems like a pretty wide reaching change. Is it definitely good across shop and subscription?

**Comment by notlee** [comment]
Maybe the layouts "example feature page" and "feature block examples" should refer to each other? It's not clear how they differ/complement each other.

**Comment by notlee** [comment]
Instead of `feature--half` maybe something like `feature--two-by-one` would make more sense? Perhaps that ship has sailed.

**Comment by notlee** [comment]
@leejordan Loving the feature blocks!

File: less/common/typography.less (hunk 1 above)
Comment (leejordan) [inline less/common/typography.less]: true, reverted

**Pull request merged**